### ISSUE TITLE
Module not found: Error: Can't resolve './EventStack' in './node_modules/semantic-ui-react/dist/es/lib/eventStack'

### ISSUE THREAD

**Issue by mironal** [Semantic-Org/Semantic-UI-React#2867]
Hi! Thanks very useful library. 😄 

## Bug Report

I got a build error in Linux (alpine linux) environment.  😭 
When in Mac environment, I don't get an error.

I think that eventStack.js must be EventStack.js.

![image](https://user-images.githubusercontent.com/537587/40883573-a55b4ed8-673b-11e8-8d6e-e0ed95e400ed.png)

### Steps

1. Add semantic-ui-react to the package.json dependencies.
2. run `yarn`
3. run `yarn build`: I use webpack & typescript.
3. I got error.

### Actual Result

I got following error.

```
ERROR in ./node_modules/semantic-ui-react/dist/es/lib/eventStack/index.js
Module not found: Error: Can't resolve './EventStack' in '/root/project/app/node_modules/semantic-ui-react/dist/es/lib/eventStack'
 @ ./node_modules/semantic-ui-react/dist/es/lib/eventStack/index.js 1:0-38 2:19-29
 @ ./node_modules/semantic-ui-react/dist/es/lib/index.js
 @ ./node_modules/semantic-ui-react/dist/es/collections/Grid/GridRow.js
 @ ./node_modules/semantic-ui-react/dist/es/index.js
 @ ./src/App.tsx
 @ ./src/index.tsx
```

### Version

0.81.0

**Comment by layershifter** [comment]
Try to reinstall deps fully, the `master` branch has a needed file, see the [folder](https://github.com/Semantic-Org/Semantic-UI-React/tree/master/src/lib/eventStack)

**Comment by mironal** [comment]
I tried to download the tar boll from  https://registry.npmjs.org/semantic-ui-react/-/semantic-ui-react-0.81.0.tgz.

And I extract it. I got `eventStack.js` file. Is this the correct file? @layershifter 

![image](https://user-images.githubusercontent.com/537587/40884126-25e30548-6749-11e8-9c07-2b6ad96f485b.png)

**Comment by ghost** [comment]
Rename  
/node_modules/semantic-ui-react/dist/es/lib/eventStack/eventStack.js 
to
/node_modules/semantic-ui-react/dist/es/lib/eventStack/EventStack.js 

fix the error

**Comment by tkw1536** [comment]
I am affected by this issue as well, it is especially annoying because this only occurs in my automated tests (which run on Travis CI / Linux), and not in my development environment (Mac OS). 

I furthermore tried the fix from @wallena3, and noticed that it did not work in commonjs environments. 
To completely fix the issue, I had to use:

```bash
mv node_modules/semantic-ui-react/dist/es/lib/eventStack/eventStack.js node_modules/semantic-ui-react/dist/es/lib/eventStack/EventStack.js

mv node_modules/semantic-ui-react/dist/commonjs/lib/eventStack/eventStack.js node_modules/semantic-ui-react/dist/commonjs/lib/eventStack/EventStack.js 
```

Having to add these lines everywhere is less than optimal, as it is no longer possible to install `semantic-ui-react` as a `package.json` dependency and manual intervention is required. 

Can this issue please be re-opened and fixed?

**Comment by mironal** [comment]
Please reopen this issue 😭  @layershifter

**Comment by layershifter** [comment]
I reopened issue, the problem needs an investigation.

**Comment by layershifter** [comment]
The [`master`](https://github.com/Semantic-Org/Semantic-UI-React/tree/master/src/lib/eventStack) branch and the [`v0.81.0`](https://github.com/Semantic-Org/Semantic-UI-React/tree/v0.81.0/src/lib/eventStack) tag has the correctly named `EventStack.js` file. I tried to perform the build locally and everything went okay.

The `v0.81.0` contains a valid file in the `src` folder, however `dist` folders contains a wrong `eventStack.js` file :sob: Merged #2870 with same problem.

An example on [codesandbox](https://codesandbox.io/s/m78y2zprlj).

@levithomason can you check, please? As I'm remember you're running builds locally.

**Comment by liangz2** [comment]
Renaming the file does solve the issue locally, but when another dev tries to work with the new version, he/she will have to perform the same fix on their machine - not a very long term fix. With that said, thanks for the suggestion, I can do my local development for now.

**Comment by mironal** [comment]
Thanks for reopen a issue.

Was the publishing process broken?
I patiently wait to fix.

**Comment by levithomason** [comment]
Hey folks, I'll release a patch for this as soon I finish investigating why the camelCased `eventStack.js` was built.  Standby.

**Comment by levithomason** [comment]
Fixed in `semantic-ui-react@0.81.1`.  The [tarball](https://registry.npmjs.org/semantic-ui-react/-/semantic-ui-react-0.81.1.tgz) also includes the properly cased `EventStack.js` file.

**Comment by liangz2** [comment]
Hi, has the NPM package been generated? I got a message saying "No matching version found for semantic-ui-react@0.81.1". Thanks.
